Scripts/Python/xOpeningSequence.py

```diff
@@ -237,6 +236,14 @@ def OnGUINotify(self,id,control,event):
                     PtGUICursorOn()
                     OrientationDlg.dialog.show()
                     PtDebugPrint("xOpeningSequence - no intro movie!!!",level=kDebugDumpLevel)
+                    if IsTutorialPath():
+                        PtFindSceneobject("OrientationPBIcon01","GUI").draw.disable()
+                        PtFindSceneobject("OrientationPBIcon02","GUI").draw.disable()
+                        PtFindSceneobject("OrientationPBIcon01Zandi","GUI").draw.enable()
+                        ptGUIControlTextBox(OrientationDlg.dialog.getControlFromIndex(3)).setStringW(PtGetLocalizedString("GUI.OrientationGUI.OrientPBTextZandi"))
```

Comment:
I think it would be better if you added a tag ID to the OrientPBText control in Max and used that here. I think you can do this with a "GUI Control Tag" component but don't remember for sure. The way you're doing things here is liable to break if someone ever re-orders the controls in either the max file or the PRP. While you're touching things in Max, you may also consider making Scene Object attributes for the icons.